index.html

```diff
@@ -4,23 +4,54 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+    <link rel="stylesheet" href="/style.css">
     <title>Document</title>
 </head>
-<body>
-    <h1>Awesome books</h1>
-    <div class="books">
-
+<body class="container-fluid w-100 d-flex flex-column align-items-center">
+  <nav class="navbar container-fluid d-flex justify-content-between">
+    <a class="link-unstyled" href="#"><img alt="logo" src="./Captura.PNG"></a>
+    <div class=" d-flex  justify-content-around">
+      <a class="nav-link" href="#" id="add">Add new</a>
+      <a class="nav-link" href="#" id="list">List</a>
+      <a class="nav-link" href="#" id="contact">Contact</a>
+    </div>
```

Comment:
- [ ] OPTIONAL
- For a well structured and cleaner HTML syntax kindly us `ul` and `li` tags for the organizing the menu items
- The right menu elements in your navbar is missing the **dividers** between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/129396485-a808f56c-878f-4743-8908-114eda779ce7.png)

**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)